index.bs

```diff
@@ -241,7 +246,13 @@ external hardware, or a combination of both.
 
 ## [RPS] ## {#conforming-relying-parties}
 
-A [=[RP]=] MUST behave as described in [[#rp-operations]] to get the security benefits offered by this specification.
+A [=[RP]=] MUST behave as described in [[#rp-operations]] to obtain the security benefits offered by this specification.
+
+## All Conformance Classes ## {#conforming-all-classes}
+
+All [=CBOR=] encoding and decoding performed by the members of the above conformance classes MUST be done using the
```

Comment:
Encoding is MUST. Decoding is SHOULD as per CTAP spec.

Reply:
thx, fixed to match language in fido-alliance/fido-2-specs#401